content/docs/user-guide/gems/reference/ros2/_index.md

```diff
@@ -0,0 +1,119 @@
+---
+linkTitle: ROS2
+title: ROS2 Gem
+description: The ROS 2 Gem helps to build robotic simulations with ROS 2 / Robot Operating System.
+toc: true
+---
+
+<!-- # O3DE ROS2 Gem -->
+
+This Gem enables users to develop robotic simulations through ROS2 tools and components.
+
+## Requirements
+
+* Ubuntu 20.04 or 22.04. Other Ubuntu versions and Linux distros could also work as long as they can support ROS 2.
+* [O3DE](https://www.o3de.org/)
+* Modern version of ROS 2. This instruction assumes that the `desktop` version is installed. Otherwise some packages might be missing. We support and tested with:
+  * [ROS 2 Galactic](https://docs.ros.org/en/galactic/Installation.html) with Ubuntu 20.04.
+
+#### Additional ros packages required
+
+Once the desired version of ROS 2 is installed, you will need to install additional ROS 2 packages. 
+Before running the following commands, make sure that the ROS 2 environment is setup. Replace \<distro\> with the ROS 2 distribution name (galactic, humble, ..) below and run the following command:
+```
+source /opt/ros/<distro>/setup.bash
+```
+
+* gazebo_msgs: `sudo apt install ros-${ROS_DISTRO}-gazebo-msgs`
+* Ackermann messages: `sudo apt install ros-${ROS_DISTRO}-ackermann-msgs`
+* Control toolbox `sudo apt install ros-${ROS_DISTRO}-control-toolbox`
+
+If a `desktop` installation of ROS 2 distro was selected, everything else should be there.
+
+Use this helpful command to install:
+
+```
+sudo apt install ros-${ROS_DISTRO}-ackermann-msgs ros-${ROS_DISTRO}-control-toolbox ros-${ROS_DISTRO}-nav-msgs ros-${ROS_DISTRO}-gazebo-msgs
+```
+
+## Features
+
+* Direct and natural support of ROS2 ecosystem:
+  * No bridges. Your simulation node will function as any other ROS2 node.
+    * This is also good for performance
+  * Easy way to include ROS2 dependencies.
+* Sensors:
+  * Sensor Component serving as a handy abstraction.
+  * Example implementations of Lidar, Camera, IMU sensors.
+    * Including a few Assets and prefabs which are ready to use. 
+* Automated handling of:
+  * Simulation time, publishing `/clock` supporting non-real time.
+  * Publishing of transformation frames (`/tf`, `/tf_static`).
+  * Validation for topic and namespace names.
+* Robot Control Component:
+  * A quick to use method of controlling your robot with Twist messages.
+  * Can be used with custom LUA scripting. 
+
+For a "feel" of these features, see an [example project](https://github.com/o3de/RobotVacuumSample) which uses this Gem to run navigation stack.
+
+<!-- ## Clone this repository
```

Comment:
It would be good to take out common instructions to make a new project from the RobotVacuumSample README and add information about Robotics template here instead of the commented content.

Chapter: creating a robotics project with the Gem using a template.

Reply:
Will add creating project from template.